modules/dev-tools/assets/js/editor/dev-tools.js

```diff
@@ -1,14 +1,102 @@
+/**
+ * @typedef {Object} Version
+ * @property {number} major1
+ * @property {number} major2
+ * @property {number} minor
+ * @property {string} build
+ */
 export default class extends elementorModules.editor.utils.Module {
-	onElementorLoaded() {
-		this.notifyDeprecated();
-	}
-
-	notifyDeprecated() {
+	notifyBackendDeprecations() {
 		// eslint-disable-next-line camelcase
 		const notices = elementor.config.dev_tools.deprecation.soft_notices;
 
 		Object.entries( notices ).forEach( ( [ key, notice ] ) => {
 			elementorCommon.helpers.softDeprecated( key, ...notice );
 		} );
 	}
+
+	/**
+	 * @param {string} version
+	 *
+	 * @return {Version}
+	 */
+	parseVersion( version ) {
+		const versionParts = version.split( '.' ),
+			versionPartsLength = versionParts.length;
+
+		if ( versionPartsLength < 3 || versionPartsLength > 4 ) {
+			throw new RangeError( 'Invalid Semantic Version string provided' );
+		}
+
+		const [ major1, major2, minor ] = versionParts,
```

Comment:
I think it should be:
```suggestion
		const [ major, minor, patch ] = versionParts,
```

Reply:
In WordPress world it `major1, major2, minor`.